ocp_resources/resource.py

```diff
@@ -50,6 +49,14 @@
 from ocp_resources.exceptions import MissingRequiredArgumentError, MissingResourceResError
 from ocp_resources.utils import skip_existing_resource_creation_teardown
 
+DEFAULT_HTTP_STATUS_CODE_TO_RETRY = urllib3.util.Retry.RETRY_AFTER_STATUS_CODES | {
+    http.HTTPStatus.INTERNAL_SERVER_ERROR.value,
+    http.HTTPStatus.GATEWAY_TIMEOUT.value,
+    http.HTTPStatus.FORBIDDEN.value,
+    http.HTTPStatus.CONFLICT.value,
```

Comment:
Usually after a  `HTTPStatus.CONFLICT`, urllib3 retry mechanism won't help. You need to update the `metadata.resourceVersion` field before retrying.

Reply:
We get ConflictError during ResourceEditor calls often. Using timeout sampler this currently works. If urllib3 retry won't work the same way, then we have a problem here. @myakove I would need your suggestion here.